source/operations_and_monitoring.rst

```diff
@@ -260,137 +274,30 @@ Ansible control host
 Control Plane Monitoring
 ========================
 
-Monasca has been configured to collect logs and metrics across the control
-plane. It provides a single point where control plane monitoring and telemetry
-data can be analysed and correlated.
+The control plane has been configured to collect logs centrally using the EFK
+stack (Elasticsearch, Fluentd and Kibana).
 
-Metrics are collected per server via the `Monasca Agent
-<https://opendev.org/openstack/monasca-agent>`__. The Monasca Agent is deployed
-and configured by Kolla Ansible.
+Monitoring of the control plane is performed by Prometheus. Metrics are
+collected by Prometheus exporters, which are either running on all hosts (e.g.
+node exporter), on specific hosts (e.g. controllers for the memcached exporter
+or monitoring hosts for the OpenStack exporter). These exporters are scrapped
+by the Prometheus server.
 
-Logging to Monasca is done via a `Fluentd output plugin
-<https://github.com/monasca/fluentd-monasca>`__.
-
-Configuring Monasca Alerts
---------------------------
+Configuring Prometheus Alerts
+-----------------------------
 
 Generating Metrics from Specific Log Messages
 +++++++++++++++++++++++++++++++++++++++++++++
 
-If you wish to generate alerts for specific log messages, you must first
-generate metrics from those log messages. Metrics are generated from the
-transformed logs queue in Kafka. The Monasca log metrics service reads log
-messages from this queue, transforms them into metrics and then writes them to
-the metrics queue.
-
-The rules which govern this transformation are defined in the logstash config
-file. This file can be configured via kayobe. To do this, edit
-``etc/kayobe/kolla/config/monasca/log-metrics.conf``, for example:
-
-.. code-block:: text
-
-   # Create events from specific log signatures
-   filter {
-     if "Another thread already created a resource provider" in [log][message] {
-       mutate {
-         add_field => { "[log][dimensions][event]" => "hat" }
-       }
-     } else if "My string here" in [log][message] {
-       mutate {
-         add_field => { "[log][dimensions][event]" => "my_new_alert" }
-       }
-    }
-
-Reconfigure Monasca:
-
-.. code-block:: text
-
-   kayobe# kayobe overcloud service reconfigure --kolla-tags monasca
+This feature from the Monasca logging and alerting pipeline must be transposed
```

Comment:
Is this a note to self? It doesn't appear to be readily doable, unfortunately.

Reply:
I'll remove for now.